src/BasisTheory.net/Tokens/Entities/Token.cs

```diff
@@ -19,6 +20,10 @@ public class Token : TokenResponse
         [JsonPropertyName("mask")]
         public dynamic Mask { get; set; }
 
+        [JsonProperty("privacy")]
+        [JsonPropertyName("privacy")]
+        public DataPrivacy Privacy { get; set; }
```

Comment:
Just curious, why do we specify property order on the parent but not here.

Reply:
Good question. The default order is `0` and the TokenResponse properties start at `-3`, so these properties all fall in the middle without explicit order.